velox/common/memory/SharedArbitrator.cpp

```diff
@@ -1221,12 +1203,12 @@ void SharedArbitrator::freeCapacity(uint64_t bytes) {
   if (FOLLY_UNLIKELY(bytes == 0)) {
     return;
   }
-  std::vector<ContinuePromise> resumes;
+  std::vector<ContinuePromise> globalArbitrationWaitResumes;
```

Comment:
Can we just use short names which is clear in the context? Thanks!

Reply:
In freeCapacity() method scope there isn't any context mentioning the resume is for global arbitration. So I changed the name to indicate. This way I don't need to refer to other locations of the file to figure out this resume is for global arbitration anymore.